troposphere/eks.py

```diff
@@ -188,6 +193,7 @@ class Nodegroup(AWSObject):
         "ReleaseVersion": (str, False),
         "RemoteAccess": (RemoteAccess, False),
         "ScalingConfig": (ScalingConfig, False),
+        "UpdateConfig": (UpdateConfig, False),
```

Comment:
Quick change for this PR, can you alphabetize this new property? Thanks!

Reply:
can you kindly run workflows once more? thanks! @markpeek

Reply:
Try this if you have the black formatter installed:
```
black troposphere/eks.py
````

Reply:
indentation issues fixed, thanks.